RedBeanPHP/Logger/RDefault/Debug.php

```diff
@@ -180,13 +202,13 @@ public function log()
 		$sql = func_get_arg( 0 );
 
 		if ( func_num_args() < 2) {
-			$bindings = array();
+		    $bindings = NULL;
 		} else {
-			$bindings = func_get_arg( 1 );
+		    $bindings = func_get_arg( 1 );
 		}
 
-		if ( !is_array( $bindings ) ) {
-			return $this->output( $sql );
+		if ( !(boolean) $bindings || !is_array( $bindings )) {
```

Comment:
This is a bit confusing. Something like this seems more understandable in my opinion:
```php
if ( !is_array( $bindings ) || empty( $bindings ) )
```